tests/benchmarks/garage/tf/algos/test_benchmark_trpo.py

```diff
@@ -105,7 +105,7 @@ def test_benchmark_trpo(self):  # pylint: disable=no-self-use
             benchmark_helper.plot_average_over_trials(
                 [baselines_csvs, garage_tf_csvs, garage_pytorch_csvs],
                 [
-                    'eprewmean', 'Evaluation/AverageReturn',
+                    'EpRewMean', 'Evaluation/AverageReturn',
```

Comment:
i think this may have been in lowercase so that it has the same column name as what baselines uses, but you should check.

Reply:
I checked the log file of trpo_mpi (openai.baseline) and their column name is EpRewMean.
(for ppo2, it is eprewmean)